scripts/run_all_scheduled_functions.py

```diff
@@ -0,0 +1,5 @@
+from bots.tasks import run_all_scheduled_functions
+
+
+def run():
+    run_all_scheduled_functions.delay()
```

Comment:
this file isn't needed anymore with `celery beat`, but can be used to trigger running all functions manually